src/entities/MovieDetails/MovieDetails.jsx

```diff
@@ -9,13 +11,17 @@ export const MovieDetails = ({ movie }) => {
         <img src={imgUrl} alt="movie" />
       </div>
       <div className="movie-content">
-        <div className="movie-title">
-          <p className="name">{movieName}</p>
-          <p className="rating">{rating}</p>
+        <div className="movie-title-wrapper">
+          <div className="movie-title">
+            <p className="name">{movieName}</p>
+            <p className="rating">{rating}</p>
+          </div>
+          <p className="movie-close" onClick={handleMovieViewClosed}>X</p>
```

Comment:
from semantic perspective, it's not a good idea to use <p> tag a button

Reply:
done